tests/integration/cattletest/core/test_resource_based_scheduling.py

```diff
@@ -95,6 +95,29 @@ def test_resource_based_scheduler(new_context, super_client):
                              'type': 'computePool'}])
 
 
+def test_delete_host(new_context, super_client):
```

Comment:
Did you verify that this test fails with the old code?
I just want to make sure that we know this test fails without the fix and passes with the fix. (You should do this with all bug fix tests actually)